kyuubi-server/src/main/scala/org/apache/kyuubi/session/KyuubiSessionManager.scala

```diff
@@ -126,6 +131,12 @@ class KyuubiSessionManager private (name: String) extends SessionManager(name) {
       setSession(handle, batchSession)
       info(s"$user's batch session with $handle is opened, current opening sessions" +
         s" $getOpenSessionCount")
+      sessionStateStore.insertBatch(
```

Comment:
shall we do insert batch inside `KyuubiBatchSessionImpl` so all actions of state are in the `KyuubiBatchSessionImpl`